src/task_run_diagnostic_reports.py

```diff
@@ -20,9 +20,18 @@ def run_diagnostic_reports(group, disease):
         return []
 
 
+def publish_report(wrapper, name, version):
+    try:
+        logging.info("Publishing report version {}-{}".format(name, version))
+        return wrapper.execute(wrapper.ow.publish_report, name, version)
+    except Exception as ex:
+        logging.exception(ex)
+        return False
+
+
 def run_reports(wrapper, config, reports):
     running_reports = {}
-    new_versions = []
+    new_versions = {}
```

Comment:
changed the task return type to include publish info, see e.g. https://github.com/vimc/montagu-task-queue/pull/8/files#diff-6cc0b9967b8b360110efd25b83751465R39